packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -287,6 +259,57 @@ class TimelineScreenBodyState extends State<TimelineScreenBody>
     );
   }
 
+  Widget _buildRecordingInfo() {
+    final recordingInstructions = Column(
+      key: TimelineScreen.recordingInstructionsKey,
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        Row(
```

Comment:
To handle small screen widths better, consider using a Wrap widget instead of a Row:

https://api.flutter.dev/flutter/widgets/Wrap-class.html

To learn more about the types of layout problems @albertusangga is working to help diagnose, try experimenting with narrow screens with the two approaches.

Also, feel free to take a look at what happens when you wrap the `Text`s in `Expanded` widgets.

Reply:
added a todo to do in a follow up cl